src/Raven.Server/Documents/Handlers/BulkInsertHandler.cs

```diff
@@ -181,6 +187,13 @@ protected override long ExecuteCmd(DocumentsOperationContext context)
                     Debug.Assert(cmd.Type == CommandType.PUT);
                     try
                     {
+                        if (SkipOverwriteIfUnchanged)
+                        {
+                            var existingDoc = Database.DocumentsStorage.Get(context, cmd.Id, DocumentFields.Data, throwOnConflict: false);
+                            if (existingDoc != null && existingDoc.Data.Equals(cmd.Document))
```

Comment:
Sanity check. This will most likely hit the:
```
            if (_isRoot && other._isRoot)
            {
                if (_size == other.Size && Memory.CompareInline(_mem, other._mem, _size) == 0)
                    return true;
            }
```

case. Is this OK?